zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +58,20 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    attributes = {
+        0x0009: ("mode", types.uint8_t, True),
+    }
+    attr_config = {0x0009: 0x01}
+
+    async def bind(self):
+        """Bind cluster."""
+        result = await super().bind()
+        await self.write_attributes(self.attr_config, manufacturer=OPPLE_MFG_CODE)
```

Comment:
Is there a variable declaration missing for `OPPLE_MFG_CODE`? I see in [other files](https://github.com/zigpy/zha-device-handlers/blob/e4d0663aa4fb2f4bd0b41825f1942e3d29893375/zhaquirks/xiaomi/aqara/opple_remote.py#L99) it's defined as `OPPLE_MFG_CODE = 0x115F`.

Reply:
I concur. adding this definition allows also the battery to be recognized and displayed!